### PR TITLE
[frontend] show right navigation menu when some manager are disabled.

### PR DESCRIPTION
**Pull request by aHenryJard**
### Proposed changes

* When a manager is disabled, the right navigation menu is hidden, so it's not possible to navigate to other page like connector.

Example before change:
Capture d'écran 2024-02-13 174456

Same screen after change:
Capture d'écran 2024-02-14 113951

### Related issues

* No issue as far as I know.

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/data/IngestionCsv.tsx [hunk 1]
```diff
@@ -109,9 +109,12 @@ const IngestionCsv = () => {
   };
   if (!platformModuleHelpers.isIngestionManagerEnable()) {
     return (
-      <Alert severity="info">
-        {t_i18n(platformModuleHelpers.generateDisableMessage(INGESTION_MANAGER))}
-      </Alert>
+      <div>
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/data/IngestionCsv.tsx]: add `className={classes.container}`

Comment (aHenryJard) [reply opencti-platform/opencti-front/src/private/components/data/IngestionCsv.tsx]: done, you are right it's better (updating screenshot...)

File: opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts [hunk 2]
```diff
@@ -178,7 +178,7 @@ describe('Indicator resolver standard behavior', () => {
 
     expect(queryResult.data?.indicator.decay_base_score).toBe(50);
     const startDecayDate: Date = queryResult.data?.indicator.decay_base_score_date;
-    expect(startDecayDate.getTime() / 1000).toBeCloseTo(new Date().getTime() / 1000, 0); // approximately now
+    expect(startDecayDate.getTime() / 10000).toBeCloseTo(new Date().getTime() / 10000, 0); // approximately now
```
Comment (Kedae) [inline opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts:181]: Why this change ?

**Comment by Kedae** [comment]
Can you rename branch as bug/***** ?

File: opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts (hunk 2 above)
Comment (aHenryJard) [reply opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts:181]: I just got a build fail on this PR because of this test that is a bit too agressive on date. So it's a flake, that I'm fixing in the way. I can do a separate PR but I prefer to have it soon in case it's distubing too much PR builds.

Capture d'écran 2024-02-14 162749

**Review by Kedae — APPROVED** [review]

**Review by SouadHadjiat — APPROVED** [review]
